README.md

```diff
@@ -89,59 +92,26 @@ public void onTasksChanged() {
     }
 }
 ```
-## Additional dependencies
-
-This project uses the Loaders framework available from Android 3.0 (API Level
-11).
-
-## Features
-
-### Complexity - understandability
-
-#### Use of architectural frameworks/libraries/tools:
 
-No external frameworks.
+## Maintaining the app
 
-#### Conceptual complexity
+Adding or amending features to this version of the app requires a similar amount of effort to the todo-mvp sample. Before modifying the sample, you should be familiar with the [Loaders API](https://developer.android.com/guide/components/loaders.html), which is a relatively complex topic.
 
-Developers need to be familiar with the Loaders framework, which is not
-trivial.
+This sample introduces two new classes, in addition to those found in todo-mvp: [`TaskLoader`](https://github.com/googlesamples/android-architecture/blob/todo-mvp-loaders/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/TaskLoader.java), and [`TasksLoader`](https://github.com/googlesamples/android-architecture/blob/todo-mvp-loaders/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/TasksLoader.java). Some aspects of the code are simpler as loaders take care of asynchronous work. The table below summarizes the amount of code used to implement this version of the app. You can compare it with similar tables provided for each of the other samples in this project.
```

Comment:
Leaving this as is for consistency with other readme files. (repetition is good in this context and for readers skimming for specific information)